python/lsst/verify/bin/dispatchverify.py

```diff
@@ -394,7 +401,11 @@ def __init__(self, args):
             message = '--user or $SQUASH_USER configuration required'
             raise RuntimeError(message)
 
-        self.api_password = args.api_password or os.getenv('SQUASH_password')
+        if args.interactive:
+            self.api_password = getpass.getpass(prompt="SQuaSH password: ")
+        else:
+            self.api_password = (args.api_password or
+                                 os.getenv('SQUASH_password'))
         if not self.test and self.api_password is None:
```

Comment:
I don't recall what `self.test` is, but I think you'd want to make the logic for opening a prompt also be bypass-able through the `self.test` flag.

Reply:
You mean bypass if `--interactive` is set and `test` is set?

Reply:
If the `--test` flag was set by a user, don't use `getpass.getpass` to request a password since that password won't be used.